Earthfile

```diff
@@ -818,3 +818,25 @@ npm-update-all:
         RUN cd $nodepath && npm update
         SAVE ARTIFACT --if-exists $nodepath/package-lock.json AS LOCAL $nodepath/package-lock.json
     END
+
+# merge-main-to-docs merges the main branch into docs-0.7
+merge-main-to-docs:
+    FROM alpine/git
+    # Ok, if you have no choice
+    ARG git_url="git@github.com:earthly/earthly"
+    GIT CLONE "$git_url" earthly
+    RUN --mount=type=secret,id=littleredcorvette-id_rsa,mode=0400,target=/root/.ssh/id_rsa \
+          ssh-keyscan github.com >> ~/.ssh/known_hosts && \
+        rm -rf earthly &&\
+        git clone "$git_url" earthly
+    WORKDIR earthly
+    RUN git config --global user.name "littleredcorvette" && \
+        git config --global user.email "littleredcorvette@users.noreply.github.com"
+    RUN git checkout docs-0.7 && \
+    git merge --no-commit --no-ff main && \
+    git status && \
+    git merge --abort
```

Comment:
ah ha..... is this for a dry-run sort of output? it was a bit confusing without a comment.

honestly maybe we can just remove it and use the single command you have below without a `git push` at the end for the initial trial?

If there's any conflicts it'll fail right?

Reply:
yeah, I think if git merge has any conflicts it will return an error.
sure, I can merge the command into 

perhaps I leave the dry run mode we have for now just so we can see `git status` and when ready I can remove it?

Reply:
discussed offline - going to keep for now and can remove when ready to get out of dry run mode